chainermn/communicators/nccl_communicator.py

```diff
@@ -0,0 +1,77 @@
+import chainer.cuda
+import math
+
+from chainermn.communicators import _base
+from chainermn.communicators import _communication_utility
+from chainermn.communicators import _memory_utility
+from chainermn import nccl
+
+
+class NcclCommunicator(_base.CommunicatorBase):
```

Comment:
We already have `chainermn.nccl.NcclCommunicator`. This adds new `NcclCommunicator` (`chainermn.communicators.NcclCommunicator`). I think it is quite confusing. Is it possible to change the name of either one?